.idea/gradle.xml

```diff
@@ -5,7 +5,7 @@
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
         <option name="delegatedBuild" value="false" />
-        <option name="testRunner" value="PLATFORM" />
+        <option name="testRunner" value="GRADLE" />
```

Comment:
to exclude files that already were included but now ignored through `.gitignore` you can do this:
- `git rm --cached <file>` to exclude a file
- `git rm -r --cached <folder>` to exclude a whole folder